app/src/main/java/com/duckduckgo/app/launch/LaunchBridgeActivity.kt

```diff
@@ -68,11 +68,11 @@ class LaunchBridgeActivity : DuckDuckGoActivity() {
         } else {
             startActivity(OnboardingActivity.intent(this))
         }
-        finish()
+        // finish()
```

Comment:
Can we kill these commented out lines? Also, if the intention is to delete it, why don't we call `finish()`?

Reply:
This was just a test, good catch!